.travis.yml

```diff
@@ -16,7 +16,7 @@ matrix:
     fast_finish: true
     include:
         - php: 7.0
-          env: SYMFONY_VERSION=2.7.*
+          env: SYMFONY_VERSION=~2.7.2 # Workaround for a Composer bug
```

Comment:
easy thing we Travis right? I think it's better than having someone for which it won't work because we failed to specify the right version in the `composer.json`